### PR TITLE
Feature/stream tx graph

### PR DESCRIPTION
**Pull request by rashtao**
stream tx support in graph vertex/edge operations at /_api/gharial

### CODE REVIEW

**Comment by rashtao** [comment]
http://jenkins01:8080/view/Java/job/arangodb-java-driver-pr/243/

File: src/test/java/com/arangodb/StreamTransactionGraphTest.java [hunk 1]
```diff
@@ -0,0 +1,396 @@
...
+        final BaseEdgeDocument value = new BaseEdgeDocument();
+        value.setFrom(v1.getId());
+        value.setTo(v2.getId());
+        return value;
+    }
+
+    @Test
+    public void getVertex() {
+        assumeTrue(isSingleServer());
+        assumeTrue(isAtLeastVersion(3, 5));
```
Comment (hkernbach) [inline src/test/java/com/arangodb/StreamTransactionGraphTest.java]: Oh, one thing:
`        assumeTrue(isAtLeastVersion(3, 5));`

^-- does this needs to be instead `isAtLeastVersion(3,5,1)` ?

Comment (rashtao) [reply src/test/java/com/arangodb/StreamTransactionGraphTest.java]: We are still discussing about which db versions to support and to test against... For the moment we only support and test against the latest patch of every minor release. Eg. atm 3.3.23, 3.4.8 and 3.5.0.

**Comment by rashtao** [comment]
http://jenkins01:8080/view/Java/job/arangodb-ANY-java-driver-matrix/290/

**Comment by rashtao** [comment]
http://jenkins01:8080/view/Java/job/arangodb-java-driver-pr/246/

**Pull request merged**